liquibase-couchbase/src/test/java/common/operators/TestCollectionOperator.java

```diff
@@ -42,4 +47,9 @@ public static JsonObject createOneFieldJson(String id, String value) {
     public static JsonObject createTestDocContent() {
         return createOneFieldJson("key", "value");
     }
+
+    public void removeAllDocuments(Scope scope) {
+        QueryOptions queryOptions = queryOptions().scanConsistency(QueryScanConsistency.REQUEST_PLUS);
+        scope.query(format("DELETE FROM %s.%s.%s", collection.bucketName(), collection.scopeName(), collection.name()), queryOptions);
```

Comment:
What is the purpose of providing `scope` as a method parameter if we don't use it? Possible mistake in `collection.scopeName()` instead of `scope.name()`? Otherwise, we don't need scope parameter in the method

Reply:
1) We need to perform custom query on collection. But we can't do this through `Collection` class. That's why we need `Scope`

2) Doesn't matter collection.scopeName() or scope.name().

Reply:
But I think we can move this `removeAllDocuments` method from `CollectionOperator` to `ClusterOperator` for example.

Reply:
Method could be moved to clusterOperator like `removeAllDocuments (Keyspace keyspace)` and use `cluster.query()` inside

Reply:
Done